app/client/src/components/shared/MapWidgets/index.js

```diff
@@ -96,6 +99,7 @@ function updateVisibleLayers(view: any, legendNode: Node) {
     'mappedWaterLayer',
     'countyLayer',
     'watershedsLayer',
+    'upstreamWatershed',
```

Comment:
For this one I think the "upstreamWatersheds" layer should be above the boundaries layer so the legend position more closely reflects the ordering it is displayed on the map.

Reply:
Changed, thanks!